.github/workflows/binaries.yml

```diff
@@ -0,0 +1,126 @@
+name: Binaries Build
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build-binaries:
+    name: Build ${{ matrix.binary }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            binary: fioctl-linux-amd64
+          - os: ubuntu-latest
+            binary: fioctl-linux-arm64
+            qemu: true
+            qemu-platform: arm64
+          # Let's build on the ubuntu to be faster
+          - os: ubuntu-latest
+            binary: fioctl-windows-amd64.exe
+          - os: macos-latest
+            binary: fioctl-darwin-amd64
+          - os: ubuntu-latest
+            binary: fioctl-linux-armv7
+            qemu: true
+            qemu-platform: linux/arm/v7
```

Comment:
@vkhoroz @doanac 
To address this one https://github.com/foundriesio/fioctl/pull/269/files#r1334637439 I added the teste here as well

Reply:
removed